airflow/models.py

```diff
@@ -3389,7 +3389,7 @@ def __init__(
         self.on_success_callback = on_success_callback
         self.on_failure_callback = on_failure_callback
 
-        self._context_manager_set = False
+        self._old_context_manager_dags = []
```

Comment:
Nit: How about just context_manager_dags since it includes the current one?

Reply:
Actually this doesn't contain the current one. It just contains what was in `_CONTEXT_MANAGER_DAG` before it is set to the current DAG. The `old` naming convention was existing, and the only difference here is making into a list.